src/cmd/initialize.go

```diff
@@ -227,3 +236,42 @@ func init() {
 
 	initCmd.Flags().SortFlags = true
 }
+
+func chooseInitPackage(args []string) (string, error) {
+	if len(args) > 0 {
+		return args[0], nil
+	}
+	var path string
+	prompt := &survey.Input{
+		Message: lang.CmdPackageChoose,
+		Suggest: func(toComplete string) []string {
+			tarPath := config.ZarfInitPackagePrefix + toComplete + "*.tar"
+			files, err := filepath.Glob(tarPath)
+			if err != nil {
+				message.Debug("Unable to glob", "tarPath", tarPath, "error", err)
+			}
+
+			zstPath := config.ZarfInitPackagePrefix + toComplete + "*.tar.zst"
+			zstFiles, err := filepath.Glob(zstPath)
+			if err != nil {
+				message.Debug("Unable to glob", "zstPath", zstPath, "error", err)
+			}
+
+			splitPath := config.ZarfInitPackagePrefix + toComplete + "*.part000"
+			splitFiles, err := filepath.Glob(splitPath)
+			if err != nil {
+				message.Debug("Unable to glob", "splitPath", splitPath, "error", err)
+			}
+
+			files = append(files, zstFiles...)
+			files = append(files, splitFiles...)
+			return files
+		},
+	}
+
+	if err := survey.AskOne(prompt, &path, survey.WithValidator(survey.Required)); err != nil {
+		return "", fmt.Errorf("package path selection canceled: %w", err)
+	}
+
+	return path, nil
```

Comment:
Added to follow the pattern laid out by `zarf package deploy`; however seeing I want to leave the room for backward-compatibility I'm not sure if there is an `optional` setting that we change it too?